helm/ovn-kubernetes/values-multi-node-zone.yaml

```diff
@@ -0,0 +1,163 @@
+# Values for ovn-kubernetes with multi-node zone interconnect
+# Requires: ovnkube-control-plane, ovnkube-zone-controller, ovnkube-node
+# enableOvnKubeIdentity need to be false 
+
+# -- list of dependent subcharts that need to be installed for the given deployment mode, these subcharts haven't been tested yet.
+tags:
+  ovn-ipsec: false
+  ovnkube-db: false
+  ovnkube-db-raft: false
+  ovnkube-master: false
+  ovnkube-node-dpu: false
+  ovnkube-node-dpu-host: false
+  ovnkube-single-node-zone: false
+  ovnkube-control-plane: true
```

Comment:
Are these 3 lines needed? iirc, only the instance we are setting to false needed to be mentioned here.

Reply:
somehow these lines are needed, otherwise the required components won't be created, this is not consistent with document, but I have no idea why it behaves like this yet.